modules/sdk-core/src/bitgo/wallet/iWallet.ts

```diff
@@ -563,6 +581,8 @@ export interface WalletData {
   type?: WalletType;
   subType?: SubWalletType;
   tokens?: Record<string, any>[];
+  nfts?: { [key: string]: NftBalance };
+  unsupportedNfts?: { [key: string]: NftBalance };
```

Comment:
what is the key? an address maybe?

Reply:
+1, a type alias `type Address = string` could add this context or a comment.

Reply:
I was thinking rather `{ [address: string]: ... }`

Reply:
Forgot you could do that, yes that also works!